### ISSUE TITLE
Output of "type docker" shouldn't show in console

### ISSUE THREAD

**Issue by glatard** [boutiques/boutiques#510]
To check if Docker is installed, the code makes a system call to execute `type docker`. The output of this command appears on the console, in stdout when Docker is installed, in stderr when it is not. This should be cleared.

## Expected Behavior

Assuming that Docker is not available.
```
$ bosh exec launch -s zenodo.1482743 '{"infile": "./data/test.nii.gz", "maskfile": "test_brain.nii.gz"}'
Shell command
bet ./data/test.nii.gz test_brain.nii.gz  
...
```

## Current Behavior
```
$ bosh exec launch -s zenodo.1482743 '{"infile": "./data/test.nii.gz", "maskfile": "test_brain.nii.gz"}'
/bin/sh: line 0: type: docker: not found
Shell command
bet ./data/test.nii.gz test_brain.nii.gz  
...
```
Note the `type:docker: not found`.

## Context (optional)
I am preparing a tutorial on Boutiques, this output message looks like an error but it's not.

**Comment by prioux** [comment]
I was just about to create a bug report for that, but GitHub's suggestions for related issues helped me avoid creating a duplicate.

**Comment by prioux** [comment]
The bug is in localExec.py

```python
    def _isDockerInstalled(self):
        return not subprocess.Popen("type docker", shell=True).wait()
```

It's brittle because it assumes the shell that will run the command is `bash`. (Not sure it exists in ksh, or csh, or plain sh...)

**Comment by gkiar** [comment]
I actually fixed this in #537 -- it's not in a release yet, but on the dev branch
